pkg/services/librarypanels/database.go

```diff
@@ -399,6 +405,30 @@ func (lps *LibraryPanelService) getLibraryPanelsForDashboardID(c *models.ReqCont
 	return libraryPanelMap, err
 }
 
+func handleFolderIDPatches(session *sqlstore.DBSession, panelToPatch *LibraryPanel, fromFolderID int64, toFolderID int64, user *models.SignedInUser) error {
+	// FolderID was not provided in the PATCH request
+	if toFolderID == -1 {
+		panelToPatch.FolderID = fromFolderID
```

Comment:
I'm wondering if the following makes for less complex/more straightforward code?
```suggestion
		toFolderID = fromFolderID
```

Reply:
I think this makes sense, but I need to update panelToPatch too.